scenarioo-docu-generation-example/src/test/resources/example/documentation/scenarioDocuExample/wikipedia-docu-example-dev/2014-04-19/build.xml

```diff
@@ -3,5 +3,6 @@
     <name>2014-04-19</name>
     <revision>F398DA3</revision>
     <date>2014-04-19T00:00:00+02:00</date>
+    <status>success</status>
```

Comment:
@adiherzog This commit to change the test data was unnecessary - the example documentaiton data should always be commited (only!) after a clean run of "scenarioo-docu-generation-example" - this fields are added by the viewer after importing the builds the first time, and it is intended to not have them in the example data for testing that the state gets automatically added by the server. But yes, I know, this is not optimal and we might have to improve that once. until we improve it we should only commit example data, when generated with a clean build.

Reply:
@adiherzog hmmm - ok, forget - it seems the tests are now generating that status for the builds always??? Why??? I think this has been changed wrongly.

Reply:
@adiherzog: I do not get it - somehwo it seems that you copied it from the "out" directory - there it is generated with the state - I not get why there is this "out"-directory - cant we change that to write to build as well and use meaningful name why this output is produced (probably testing the import)

Reply:
@adiherzog ahhhrgh - okay - think now I understand - sorry - my first comment was right - the version in the out directory is always the commited one (gets copied during build somehow) - will now copy again the version from build and then it should be okay again.